### DISCUSSION TITLE
Failed to run on github action (`open /etc/cni/tuning/allowlist.conf: permission denied`)

### DISCUSSION THREAD

**Discussion by wzshiming** [containerd/nerdctl#2084] [Q&A] [answered]
### Description

``` console
nerdctl run --rm registry.k8s.io/etcd:3.4.13-0 etcd --version: exit status 1
time="2023-02-02T10:17:03Z" level=fatal msg="failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: time=\"2023-02-02T10:17:03Z\" level=fatal msg=\"failed to call cni.Setup: plugin type=\\"tuning\\" failed (add): open /etc/cni/tuning/allowlist.conf: permission denied\"
```

### Steps to reproduce the issue

1. Install nerdctl on github action
2. Run

containernetworking/plugins#826

### Describe the results you received and expected

normal work

### What version of nerdctl are you using?

nerdctl v1.2.0

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

**Comment by AkihiroSuda** [comment, marked as answer]
https://github.com/containerd/nerdctl/releases/tag/v1.2.0
> **Note**
>
> **Upgrade notice**
>
> The CNI plugins package bundled with the `nerdctl-full` distribution is upgraded to v1.2.0 in this release.
>
> Some users of rootless mode may face an error like `open /etc/cni/tuning/allowlist.conf: permission denied` when `/etc/cni` is created with the `0700` permission on the host.
>
> In this case, run `sudo chmod 0755 /etc/cni` on the host.
